apps/store/themes/store/code/search-history.hbs

```diff
@@ -0,0 +1,6 @@
+<script type="text/javascript">
+    if(!store.store){
```

Comment:
Thanks , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/842/commits/6d963a2c9f0c71c47b7d90a3d6f64feedbf5163d)